uniffi_bindgen/src/bindings/swift/templates/macros.swift

```diff
@@ -157,15 +157,15 @@ v{{- field_num -}}
     {%- endfor %}
 {%- endmacro %}
 
-{%- macro async(func) %}
+{%- macro is_async(func) %}
```

Comment:
The renaming for this macro is because `rinja` doesn't allow to name item from Rust keywords. Same for `try` below.